docs/api/objects.rst

```diff
@@ -109,6 +110,58 @@ Connection information
             .. __: https://www.postgresql.org/docs/current/multibyte.html
 
 
+.. _capabilities:
+
+Libpq capabilities information
+------------------------------
+
+.. autoclass:: Capabilities
+
+    An instance of this object is normally exposed by the module as the object
+    `psycopg.capabilities`.
+
+    Every feature check is implemented by an `!has_SOMETHING()` method. All
+    the methods return a boolean value stating if the capability is supported,
+    which can be used by a program to degrade gracefully::
+
+        if psycopg.capabilities.has_pipeline(check=True)
```

Comment:
`check=False` (or omitted) in this case, no?

Reply:
Yes, it's a mistake. Fixed, thank you